cmd/arrow/batches/arrow_batches.go

```diff
@@ -0,0 +1,153 @@
+package main
+
+import (
+	"context"
+	"database/sql"
+	"database/sql/driver"
+	"flag"
+	"fmt"
+	"github.com/apache/arrow/go/v12/arrow/array"
+	sf "github.com/snowflakedb/gosnowflake"
```

Comment:
Could you please move this import to the bottom of the list.

Reply:
Done.